QQE-378.md

```diff
@@ -0,0 +1,44 @@
+# QQE-378 - HTTP module / REST use-cases - Development topic compression Brotli4j
+## Links
+Jira : [QQE-378](https://issues.redhat.com/browse/QQE-378)
+
+PR:  TODO
+## Scope of the testing
+ - Integration of the Brotli4J compression library with Quarkus, particularly in the context of REST endpoints using RestEasy Reactive.
+ - The test coverage needs to be defined for both JVM and native mode.
```

Comment:
> defined

I think you define it here, so you are not making a plan for a plan. IMO you should just define it here, not say it needs to be defined. Maybe wrong words?